sequencing-server/src/lib/codegen/CommandTypeCodegen.ts

```diff
@@ -57,7 +57,7 @@ export const Hardwares = {\n${dictionary.hwCommands
     .map(hwCommands => `\t\t${hwCommands.stem}: ${hwCommands.stem},\n`)
     .join('')}};
 
-Object.assign(globalThis, { A:A, R:R, E:E, C:Object.assign(Commands, STEPS), Sequence}, Hardwares, Immediates);
+Object.assign(globalThis, { A:A, R:R, E:E, C:Object.assign(Commands, STEPS), Sequence, VARIABLE, BUILD_LOCALS, BUILD_PARAMETERS, MAP_VARIABLES}, Hardwares, Immediates);
```

Comment:
How do we feel about prefacing our functions with an identifier to distinguish them from our other objects?
Something like `functions.BUILD_PARAMETER(...)`?

Reply:
Are you think the user will type `functions` and get a list of available eDSL functions in the content assit?

Reply:
I honestly was thinking about something similar recently and was more on the fence if users would want to have to type more, which seems like we are trying to limit that as much as possible. It might be a good idea to have a discussion about it with the team cause it will help out new users.

I was also thinking about being more explicit so you exactly what you are typing

```ts
eDSL.step.<c/a/e>.<command>
eDSL.step.ground_block
eDSL.step.ground_event

eDSL.hdw.<hardware_command>

eDSL.immediate.<commands>

eDSL.functions.build_locals
eDSL.functions.map_variables
eDSL.functions.time.<time helper functions>
```

Reply:
maybe typing `eDSL` is too repetitive.